translationfiles/de/cookbook.po

```diff
@@ -0,0 +1,214 @@
+"Language: de_DE\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Lokalize 19.04.3\n"
+
+#: /srv/http/nc/apps/cookbook/js/script.js:91
+msgid "Path to your recipe collection"
+msgstr "Pfad zur Rezept-Datenbank"
+
+#: /srv/http/nc/apps/cookbook/js/script.js:106
+msgid "Could not set recipe folder to {path}"
+msgstr "Konnte den Pfad zu den Rezepten auf {path} setzen"
+
+#: /srv/http/nc/apps/cookbook/js/script.js:131
+msgid "Please pick a recipe"
+msgstr "Bitte ein Rezept auswählen"
+
+#: /srv/http/nc/apps/cookbook/js/script.js:152
+msgid "Could not load recipe"
+msgstr "Konnte Rezept nicht laden"
+
+#: /srv/http/nc/apps/cookbook/js/script.js:232
+msgid "Could not update recipe"
+msgstr "Konnte Rezept nicht aktualisieren"
+
+#: /srv/http/nc/apps/cookbook/js/script.js:278
+msgid "Could not add recipe"
+msgstr "Konnte Rezept nicht hinzufügen"
+
+#: /srv/http/nc/apps/cookbook/js/script.js:300
+msgid "Could not rebuild recipe index."
+msgstr "Konnte den Rezept-Index nicht neu aufbauen."
+
+#: /srv/http/nc/apps/cookbook/js/script.js:315
+msgid "Are you sure you want to delete this recipe?"
+msgstr "Soll das Rezept auch wirklich gelöscht werden?"
+
+#: /srv/http/nc/apps/cookbook/js/script.js:331
+msgid "Failed to delete recipe"
+msgstr "Löschen des Rezepts ist fehlgeschlagen"
+
+#: /srv/http/nc/apps/cookbook/js/script.js:362
+msgid "Failed to fetch recipes"
+msgstr "Laden des Rezepts ist fehlgeschlagen"
+
+#: /srv/http/nc/apps/cookbook/specialAppInfoFakeDummyForL10nScript.php:2
+msgid "Cookbook"
+msgstr "Kochbuch"
+
+#: /srv/http/nc/apps/cookbook/specialAppInfoFakeDummyForL10nScript.php:3
+msgid "An integrated cookbook using schema.org JSON files as recipes"
+msgstr ""
+"Ein integriertes Kochbuch, das schema.org JSON Dateien als Speicher für"
+" Rezepte nutzt"
+
+#: /srv/http/nc/apps/cookbook/specialAppInfoFakeDummyForL10nScript.php:4
+msgid ""
+"A library for all your recipes. It uses JSON files following the schema.org "
+"recipe format. To add a recipe to the collection, you can paste in the URL "
+"of the recipe, and the provided web page will be parsed and downloaded to "
+"whichever folder you specify in the app settings."
+msgstr ""
+"Eine Bibliothek aller Rezepte. Sie nutzt JSON Dateien, die dem schema.org"
+" Standard für Rezepte folgen. "
+"Um weitere Rezepte zu der Sammlung hinzuzufügen, kann die URL des Rezepts"
+" eingefügt werden, so dass die Seite heruntergeladen, analysiert und in einem"
+" benutzerspezifizierten Verzeichnis abgelegt wird."
+
+#. TRANSLATORS The name of the recipe
+#: /srv/http/nc/apps/cookbook/templates/content/edit.php:3
+msgid "Name"
+msgstr "Name"
+
+#: /srv/http/nc/apps/cookbook/templates/content/edit.php:8
+msgid "Image URL"
+msgstr "Bild-URL"
+
+#: /srv/http/nc/apps/cookbook/templates/content/edit.php:13
+msgid "Keywords (comma-separated)"
+msgstr "Schlagworte (mit Kommata abgetrennt)"
```

Comment:
```suggestion
msgstr "Schlagworte (durch Kommas getrennt)"
```

Reply:
@clesmian The Plural of Komma is Kommata, right? So it would be `durch Kommata getrennt`.

Reply:
According to duden.de both is correct. I guess "kommagetrennt" would also be correct and probably closer to the original.

To be honest I don't know which version is best.